CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 [//]: # (comment: Don't forget to update statsd/telemetry.go:clientVersionTelemetryTag when releasing a new version)
 
+# 4.8.0 / 2021-06-14
+
+* [BETA][IMPROVEMENT] Reduce aggregation default window to 2s to reduce reduce sampling aliasing. See [#199][]
+* [IMPROVEMENT] EOL terminate all metrics. See [#198][]
```

Comment:
Could we expand a bit so people have more context

Something like `Automatically adding a "\n" after each metrics in order to [...]. This is made to work with the option <something> available in the Datadog Agent version X.Y.Z and above`.
WDYT ?